rkt/pubkey/pubkey.go

```diff
@@ -157,6 +161,9 @@ func (m *Manager) metaDiscoverPubKeyLocations(prefix string) ([]string, error) {
 	if m.InsecureAllowHTTP {
 		insecure = insecure | discovery.InsecureHttp
```

Comment:
Merged in the spec: https://github.com/appc/spec/pull/561
But not in any appc/spec release.